### PR TITLE
added all contributers in readme

### PR DESCRIPTION
**Pull request by Ayush7614**
Added all contributers in readme

### CODE REVIEW

**Comment by Ayush7614** [comment]
cc @nehagup @slayerjain

**Comment by nehagup** [comment]
@Ayush7614 Is there any way to do this automatically? Right now my understanding is, you've added these entries manually.

**Comment by Ayush7614** [comment]
> @Ayush7614 Is there any way to do this automatically? Right now my understanding is, you've added these entries manually.

No for manually i have doing previously
image

**Comment by Ayush7614** [comment]
Like This

**Comment by slayerjain** [comment]
@Ayush7614 better solution would have be if this feature is available in contrib.rocks  - https://github.com/lacolaco/contributors-img/issues/1036

**Comment by nehagup** [comment]
@Ayush7614 Looks like the changes vanished :(

**Comment by Ayush7614** [comment]
cc @nehagup

File: README.md [hunk 1]
```diff
@@ -155,11 +155,34 @@ We'd love to collaborate with you to make Keploy great. To get started:
...
+<td align="center"><a href="https://github.com/namantaneja167"><img src="https://avatars.githubusercontent.com/u/42579074?v=4" width="100px;" alt=""/><br /><sub><b>Naman Taneja</b></sub></a><br /><a href="#contributer-namantaneja167" title="Contributer">🚧</a></td>
+<td align="center"><a href="https://github.com/rajatsharma"><img src="https://avatars.githubusercontent.com/u/13231434?v=4" width="100px;" alt=""/><br /><sub><b>Rajat Sharma</b></sub></a><br /><a href="#contributer-rajatsharma" title="Contributer">🚧</a></td>
+<td align="center"><a href="https://github.com/nwf03"><img src="https://avatars.githubusercontent.com/u/36502791?v=4" width="100px;" alt=""/><br /><sub><b>Nawaf</b></sub></a><br /><a href="#contributer-nwf03" title="Contributer">🚧</a></td>
+<td align="center"><a href="https://github.com/praneshr"><img src="https://avatars.githubusercontent.com/u/10805204?v=4" width="100px;" alt=""/><br /><sub><b>Pranesh Ravi</b></sub></a><br /><a href="#contributer-praneshr" title="Contributer">🚧</a></td>
```
Comment (nehagup) [inline README.md:180]: I couldn't find these users contributions to Keploy. Where did you get this from, please help. 🤔

File: README.md [hunk 2]
```diff
@@ -155,11 +155,34 @@ We'd love to collaborate with you to make Keploy great. To get started:
 [![Generic badge](https://img.shields.io/badge/Youtube-teal.svg?style=for-the-badge)](https://www.youtube.com/channel/UC6OTg7F4o0WkmNtSoob34lg)
 [![Generic badge](https://img.shields.io/badge/Twitter-blue.svg?style=for-the-badge)](https://twitter.com/Keployio)
 
-## 📌 Our valuable Contributors👩‍💻👨‍💻 :
```
Comment (nehagup) [inline README.md:158]: Please keep this line. And in the next line what is [emoji key] for?

**Review by nehagup — CHANGES_REQUESTED** [review]
@Ayush7614 the list of contributors added doesn't seem correct :(

**Review by nehagup — APPROVED** [review]

**Comment by nehagup** [comment]
IMO we added everyone. Please feel free to add more contributors if missed any.

**Pull request merged**